pages/workflow/PKPWorkflowHandler.php

```diff
@@ -330,8 +332,8 @@ public function index($args, $request)
             'components' => [
                 $contributorsListPanel->id => $contributorsListPanel->getConfig(),
                 $citationsForm->id => $citationsForm->getConfig(),
-                $publicationLicenseForm->id => $publicationLicenseForm->getConfig(),
-                $titleAbstractForm->id => $titleAbstractForm->getConfig(),
+                $publicationLicenseForm->id => $this->getLocalizedForm($publicationLicenseForm, $submission, $submissionContext),
```

Comment:
context needed as parameter, here and below?

Reply:
Same question here as above